projects/kit/components/files/file/file.component.ts

```diff
@@ -146,8 +146,13 @@ export class TuiFileComponent {
             return file.src;
         }
 
-        // TODO: iframe warning
-        if (file instanceof File && file.type && file.type.startsWith('image/')) {
+        if (
+            globalThis.File &&
```

Comment:
Shouldn't we use `WINDOW`? Also there's no need for TODO warning if it's accessed through window.

Reply:
fixed ✅